Cairo Desktop/CairoDesktop.Localization/DisplayString.cs

```diff
@@ -20,6 +20,7 @@ public class DisplayString
             new KeyValuePair<string, string>("Svenska", "sv_SE"),
             new KeyValuePair<string, string>("Turkish (Türkçe)", "tr_TR"),
             new KeyValuePair<string, string>("한국어", "ko_KR")
```

Comment:
Could you please add a comma to the end of this line to fix the build error?

Reply:
Done.